cmake/setup_KWIVER.sh.in

```diff
@@ -3,4 +3,4 @@
 # https://github.com/Kitware/seal-tk/blob/master/LICENSE for details.
 
 export KWIVER_PLUGIN_PATH=\
-"@PROJECT_BINARY_DIR@/lib/kwiver/modules:$KWIVER_PLUGIN_PATH"
+"@sealtk_BUILD_LIBDIR@/kwiver/modules${KWIVER_PLUGIN_PATH:+:$KWIVER_PLUGIN_PATH}"
```

Comment:
Please explain the purpose of the new variable expansion of `KWIVER_PLUGIN_PATH`. This syntax is not obvious, and does not appear to be present in a POSIX shell.

Reply:
```
${parameter:+word}
    Use Alternate Value.  If parameter is null or unset, nothing is
    substituted, otherwise the expansion of word is substituted.
```

As used here, prevents appending an empty entry to the path list.

I don't have an actual Bourne shell (`/bin/sh` → `bash`). What shell are you using that doesn't like it? What do you get for `FOO=foo; echo ${FOO:+bar}`?

Reply:
OK, I see. The two colons were confusing me. The first colon is to invoke the variable syntax, and the second is what actually goes into the string. I haven't actually tried this yet, but I couldn't find it in the shell documentation. I will try it when I get a chance, but it looks good so far.